index.html

```diff
@@ -303,13 +303,12 @@ <h4>Game streaming</h4>
 transported using RTP or RTCDataChannel.</p>
 </section> 
 <section id="auction">
-   <h4>Low latency Broadcast with Fanout</h4>
-   <p>There are streaming applications that require large scale as well as low latency.
-   Examples include sporting events, church services, webinars and company 'Town Hall' meetings.
-   Live audio, video and data is sent to thousands (or even millions) of recipients.
-   Limited interactivity may be supported, such as allowing authorized participants to ask
-   questions at a company meeting. Both the media sender and receivers may be behind a NAT.
-   P2P relays may be used to improve scalability, potentially using different transport than
+   <h4>Ultra Low latency Broadcast with Fanout</h4>
+   <p>There are streaming applications that require large scale as well as ultra low latency (glass-glass latency less than 500ms).
+   Examples include auctions, betting and financial news. Live audio, video and data is sent to
```

Comment:
remove betting due to some people who don't feel like mentioning things like gambling.

Reply:
Done.